cli/src/code_coverage/prvsnr-coverage.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+import argparse
+import subprocess
+import sys
+import os.path
+import shlex
+
+def _run_command(cmd):
```

Comment:
Prefer calling it to `_run` or `_execute`.

Reply:
Addressed